src/cast_core/src/CollectionScanner.cpp

```diff
@@ -222,7 +222,7 @@ auto evaluatePipeline(CollectionScanner::PhaseConfig* config) {
     auto expr = config->aggregatePipelineExpr->evaluate();
     bsoncxx::array::view pipeline{expr.view()["array"].get_array().value};
     mongocxx::pipeline pl{};
-    pl.append_stages(std::move(pipeline));
+    pl.append_stages(pipeline);
```

Comment:
Is this from a warning? The  mongocxx driver tests have a [std::move](https://github.com/mongodb/mongo-cxx-driver/blob/7006baba22cfdcd43a45b7298d3f66ec5f67c25b/src/mongocxx/test/spec/operation.cpp#L64-L70)?

Reply:
Ya it's a clang-tidy warning:

> Clang-Tidy: Std::move of the variable 'pipeline' of the trivially-copyable type 'bsoncxx::array::view' has no effect; remove std::move()

I don't know enough about clang tooling to turn these on in CI/lint, but CLion helpfully points them out. I tried to remove as many as possible.

Reply:
I don't have any `.clang-tidy` configs afaict. Do you have the Inspection enabled?

<img width="1094" alt="Screen Shot 2022-05-16 at 1 18 59 PM" src="https://user-images.githubusercontent.com/119094/168648295-ad9c61be-b503-422c-9161-a0a6f0bfcc2b.png">